listenbrainz/mbid_mapping/docker/crontab

```diff
@@ -3,3 +3,10 @@
 
 # Run the huesound color sync hourly
 10 * * * * listenbrainz /usr/local/bin/python /code/mapper/manage.py update-coverart >> /code/mapper/lb-cron.log 2>&1
+
+
+# Build the mb metadata cache from scratch on first thursday of every month, MB dumps run on Wed and Sat so avoid those days
+0 15 * * 4 listenbrainz [ $(date +\%d) -le 7 ] && /usr/local/bin/python /code/mapper/manage.py cron-build-mb-metadata-cache >> /code/mapper/lb-cron.log 2>&1
```

Comment:
Should we use the same log file for all cron jobs, or since we have an entire folder here, could we have a separate log file per process?

Reply:
Separate logs sound good.

Reply:
Done.